### PR TITLE
dev, 3.0: Add rolling-update and backup configuration documents for tidb in k8s

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What is changed, added or deleted? 
Add rolling-update and backup configuration documents for tidb in k8s

### Which version does your change affect? 
`dev` `3.0`

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/how-to/upgrade/tidb-in-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
+
+> **Note：**
+>
+> - Changing `enableConfigMapRollout` variable against a running cluster will trigger an rolling-update of PD, TiKV, TiDB servers even if there's no configuration change.
```
Comment (TomShawn) [inline v3.0/how-to/upgrade/tidb-in-kubernetes.md]:
```suggestion
> - Changing the `enableConfigMapRollout` variable against a running cluster will trigger a rolling update of PD, TiKV, TiDB servers even if there's no change to the configuration.
```

File: v3.0/how-to/upgrade/tidb-in-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+> - currently, changing PD's scheduler and replication configurations(the configuration key under `[scheduler]` and `[replication]` of the PD config file) after cluster creation has no effect. You have to configure these variables via `pd-ctl` after the cluster creation, see: [pd-ctl](/reference/tools/pd-control.md).
```
Comment (TomShawn) [inline v3.0/how-to/upgrade/tidb-in-kubernetes.md]:
```suggestion
> - Currently, PD's `scheduler` and `replication` configurations (the configuration key under `[scheduler]` and `[replication]` of the PD configuration file) can not be upgraded automatically.  You must upgrade them manually via `pd-ctl`. See [pd-ctl](/reference/tools/pd-control.md) for reference.
```

**Comment by aylei** [comment]
@TomShawn Thanks for your patient review!

File: dev/how-to/upgrade/tidb-in-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+title: Rolling Update TiDB in Kubernetes
+category: how-to
```
Comment (lilin90) [inline dev/how-to/upgrade/tidb-in-kubernetes.md]: Please refer to other documents and also add a `summary` line (using a complete sentence) after the `title` line.

File: v3.0/reference/configuration/tidb-in-kubernetes/backup-configuration.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+---
+title: TiDB in Kubernetetes Backup Configuration
+category: reference
```
Comment (lilin90) [inline v3.0/reference/configuration/tidb-in-kubernetes/backup-configuration.md]: Also add a summary. Thanks!

File: dev/how-to/upgrade/tidb-in-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+    ```shell
+    watch kubectl -n ${namespace} get pod -o wide
+    ```
+
+## Change TiDB cluster Configuration
+
+By default, the change of configuration files is not applied to the TiDB cluster automatcially. The servers load the new configuration file when being restarted.
+
+You can enable rolling update on configuration change by setting `enableConfigMapRollout` to `true` in the `values.yaml`. The steps are as follows:
```
Comment (DanielZhangQD) [inline dev/how-to/upgrade/tidb-in-kubernetes.md]: Suggest setting `enableConfigMapRollout` to true by default in values.yaml, it's not reasonable to request customer to delete pod manually after config change and helm upgrade.

File: dev/reference/configuration/tidb-in-kubernetes/backup-configuration.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+### `clusterName`
+
+- The name of the target cluster
+- Default: "demo"
+- The name of the TiDB cluster from which data is backed up or to which data is restored
+
+### `name`
+
+- The name of the backup
+- Default: "fullbackup-${date}". `date` is the starting time of the backup, which is accurate to minute.
```
Comment (DanielZhangQD) [inline dev/reference/configuration/tidb-in-kubernetes/backup-configuration.md]: what's the format of ${date}? An example? And change to `${}` to `<>`?

**Comment by aylei** [comment]
@DanielZhangQD @lilin90 @tomdewan I've addressed all the review comments, PTAL again.

**Pull request merged**